.all-contributorsrc

```diff
@@ -1,200 +1,142 @@
 {
-  "files": [
```

Comment:
it's unfortunate that your prettify has different settings than others? We should maybe eventually fix that at some point but 🤷‍♀️